### PR TITLE
show menu based on buildtool

### LINKED ISSUE #555: Extension menu should be contextual to the selected buildtool
**Description**
Only show commands which are related to the selected build (and generic) tool in the extension menu:
image

Add `get the bom` for gradle.

### PR DESCRIPTION
**Pull request by phillip-kruger**
contextmenu

Fix #555

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

**Comment by phillip-kruger** [comment]
Do you know what these commands is for gradle and gradle kotlin ?

**Comment by ia3andy** [comment]
@phillip-kruger what are u using to make those really nice gif?

File: library/components/extensions-picker/extension-more-dropdown.tsx [hunk 1]
```diff
@@ -19,9 +20,14 @@ export function ExtensionMoreDropdown(props: ExtensionMoreDropdownProps) {
   const addMvnExt = `./mvnw quarkus:add-extension -Dextensions="${props.id}"`;
   const addCliExt = `quarkus ext add ${props.id}`;
   const addGradleExt = `./gradlew addExtension --extensions="${props.id}"`;
+  const gradleDep = `./gradlew implementation "${props.id}"`;
+  const gradleBom = `./gradlew implementation enforcedPlatform("${gav}")`;
+  const gradleDepKot = `./gradlew implementation("${props.id}")`;
+  const gradleBomKot = `./gradlew implementation(enforcedPlatform("${gav}"))`;
```
Comment (ia3andy) [inline library/components/extensions-picker/extension-more-dropdown.tsx:26]:
```suggestion
  const gradleBomKot = `implementation(enforcedPlatform("${gav}"))`;
```

File: library/components/extensions-picker/extension-more-dropdown.tsx [hunk 2]
```diff
@@ -19,9 +20,14 @@ export function ExtensionMoreDropdown(props: ExtensionMoreDropdownProps) {
   const addMvnExt = `./mvnw quarkus:add-extension -Dextensions="${props.id}"`;
   const addCliExt = `quarkus ext add ${props.id}`;
   const addGradleExt = `./gradlew addExtension --extensions="${props.id}"`;
+  const gradleDep = `./gradlew implementation "${props.id}"`;
```
Comment (ia3andy) [inline library/components/extensions-picker/extension-more-dropdown.tsx:23]:
```suggestion
  const gradleDep = `implementation "${props.id}"`;
```

File: library/components/extensions-picker/extension-more-dropdown.tsx [hunk 3]
```diff
@@ -19,9 +20,14 @@ export function ExtensionMoreDropdown(props: ExtensionMoreDropdownProps) {
...
+  const gradleBom = `./gradlew implementation enforcedPlatform("${gav}")`;
+  const gradleDepKot = `./gradlew implementation("${props.id}")`;
```
Comment (ia3andy) [inline library/components/extensions-picker/extension-more-dropdown.tsx:25]:
```suggestion
  const gradleDepKot = `implementation("${props.id}")`;
```

**Comment by phillip-kruger** [comment]
Ok cool. Fixed

**Pull request merged**